Libraries/Text/Text.js

```diff
@@ -90,6 +91,10 @@ const Text = React.createClass({
      */
     onLongPress: React.PropTypes.func,
     /**
+     * Lets the user select text, to use the native copy and paste functionality.
```

Comment:
Added the `@platform android`, thanks for the reminder. But I'm confused what you mean by `selectableAndroid`?